dist/src/main/resources/application.properties

```diff
@@ -47,4 +47,5 @@ spring.autoconfigure.exclude=\
   org.springframework.boot.actuate.autoconfigure.security.reactive.ReactiveManagementWebSecurityAutoConfiguration, \
   org.springframework.boot.autoconfigure.elasticsearch.ElasticsearchRestClientAutoConfiguration, \
   org.springframework.boot.autoconfigure.web.reactive.function.client.ClientHttpConnectorAutoConfiguration, \
-  org.springframework.boot.autoconfigure.web.reactive.function.client.WebClientAutoConfiguration
+  org.springframework.boot.autoconfigure.web.reactive.function.client.WebClientAutoConfiguration, \
+  org.springframework.boot.autoconfigure.ldap.LdapAutoConfiguration
```

Comment:
❓ What does this do? Why did it failed without this? Do you know?

Reply:
It starts an LDAP client that fails to connect to LDAP because obviously there is no LDAP configured to run 😄 The health/ready endpoint returned always `down` since the LDAP client "integrated" there as well. That way, all tests were failing that were waiting, for example, that the gateway is up. But also independent from the test cases, Zeebe would return `down` when keeping letting Spring auto configure the LDAP client.